lib/ansible/modules/web_infrastructure/letsencrypt.py

```diff
@@ -159,6 +159,17 @@
     required: false
     default: true
     version_added: 2.5
+  deactivate_authzs:
+    description:
+      - "Deactivate authentication objects (authz) after issuing a certificate,
+         or when issuing the certificate failed."
+      - "Authentication objects are bound to an account key and remain valid
+         for a certain amount of time, and can be used to issue certificates
+         without having to re-authenticate the domain. This can be a security
+         concern. "
```

Comment:
whitespace after .

Reply:
Whoops. I'll fiix that in the next PR or so :)